staging/godeps-json-updater.go

```diff
@@ -28,9 +28,12 @@ import (
 var (
 	godepsFile           = flag.String("godeps-file", "", "absolute path to Godeps.json")
 	clientRepoImportPath = flag.String("client-go-import-path", "", "import path to a version of client-go, e.g., k8s.io/client-go/1.4")
-	prefixes             = flag.StringSlice("ignored-prefixes", []string{"k8s.io/"}, "any godep entry prefixed with the ignored-prefix will be deleted from Godeps.json")
+	ignoredPrefixes      = flag.StringSlice("ignored-prefixes", []string{"k8s.io/"}, "any godep entry prefixed with the ignored-prefix will be deleted from Godeps.json")
```

Comment:
I didn't see these two comments because they were folded. I slightly prefer current names since they are shorter. Users will be force to read the help message. wdyt?